targets/DeployDependencies.proj

```diff
@@ -20,17 +20,40 @@
           SkipUnchangedFiles="true"           
           />
   </Target>
+  
+  <!-- Copy content of BuildTools package to deployment folders -->
+  <Target Name="CopyBuildTools">
+    <PropertyGroup>
+      <BuildToolsVersionFile>$(RepoRoot)BuildToolsVersion.txt</BuildToolsVersionFile>
+    </PropertyGroup>
+    <ReadLinesFromFile File="$(BuildToolsVersionFile)">
+      <Output TaskParameter="Lines" ItemName="BuildToolsVersion" />
+    </ReadLinesFromFile>
+    <PropertyGroup>
+      <BuildToolsVersion>@(BuildToolsVersion)</BuildToolsVersion>
+    </PropertyGroup>
+    <ItemGroup>
+      <BuildToolsFiles Include="$(ToolPackagesDir)Microsoft.DotNet.BuildTools\$(BuildToolsVersion)\lib\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BuildToolsFiles)"
+          DestinationFolder="$(DeploymentDir)"
+          SkipUnchangedFiles="true"
+          />
+    <Copy SourceFiles="@(BuildToolsFiles)"
+          DestinationFolder="$(TestDeploymentDir)"
+          SkipUnchangedFiles="true"
+          />
+  </Target>
 
   <Target Name="CopyPackageContent">
     <ItemGroup>
       <!-- We want the list of files *after* restore, which happens
            early in build, so this item can't be defined outside of a
            target. -->
-      <NuspecFilesInPackagesDir Include="$(PackagesDir)\**\*.nupkg" />
-      <PackagesFoldersInPackagesDir Include="@(NuspecFilesInPackagesDir->'%(RootDir)%(Directory)')" />
+      <PackagesFoldersInPackagesDir Include="$(PackagesDir)\Microsoft.Portable.Targets\0.1.1-dev\" />
```

Comment:
This is the only place this number will occur in a build file, so factoring it out doesn't reduce duplication.  Plus, this will go away entirely once we start using a version of .NET CLI which supports contentFiles.